addons/danger/functions/fnc_brainVehicle.sqf

```diff
@@ -57,6 +57,12 @@ private _attack = _cause in [DANGER_ENEMYDETECTED, DANGER_ENEMYNEAR, DANGER_HIT,
 private _artillery = _vehicle getVariable [QEGVAR(main,isArtillery), getNumber (configOf _vehicle >> "artilleryScanner") > 0];
 if (_artillery) exitWith {
     _vehicle setVariable [QEGVAR(main,isArtillery), true];
+
+    // enemies within 12-30m may cause crew to disembark!
+    if (_attack && {_dangerCausedBy distanceSqr _vehicle < (144 + random 324)} && {currentCommand _unit isEqualTo ""}) then {
```

Comment:
When using the squared distance check, don't you need to compare it to a squared distance as well. so 144^2?

Reply:
144 is 12 * 12 

;)